features/step_definitions/advocate_new_claim_steps.rb

```diff
@@ -411,6 +411,32 @@
   create(:fee, :misc,  claim: claim, quantity: 2, amount: 5.0)
 end
 
+Given(/^I am on the new claim page with Daily Attendance Fees in place$/) do
+  create(:fee_type, :basic, description: 'Basic Fee', code: 'BAF')
+  create(:fee_type, :basic, description: 'Daily attendance fee (3 to 40)', code: 'DAF')
+  create(:fee_type, :basic, description: 'Daily attendance fee (41 to 50)', code: 'DAH')
+  create(:fee_type, :basic, description: 'Daily attendance fee (51+)', code: 'DAJ')
+  visit new_advocates_claim_path
+end
+
+When(/^I fill in the trial details$/) do
+  select2('Trial', from: 'claim_case_type_id')
+  fill_in 'claim_actual_trial_length', with: 1
```

Comment:
cool, you're right and, in fact, i had written step that can be used previously - will apply